vllm_hpu_extension/ops.py

```diff
@@ -360,6 +366,37 @@ def forward(self, hidden_states, w1, w2, score, topk):
 
         return final_hidden_states.view(-1, D)
 
+
+class DynamicFusedMOE(torch.nn.Module):
+
+    def __init__(self, num_total_experts):
+        super().__init__()
+        self.num_total_experts = num_total_experts
+
+    def forward(self, hidden_states, w1, w2, score, topk):
+        htorch.core.mark_step()
```

Comment:
What is the reason for mark step here?

Reply:
I only moved the change from vllm-fork, because the location of ops changed in 1.19.0 It was already merged on 1.18.0
@jbyczkow , can you answer this question?